src/App.js

```diff
@@ -5,14 +5,61 @@ import BoardCard from "./BoardCard";
 import { initDeck } from "./initDeck.js";
 import { sideDeck } from "./sideDeck";
 
-export default function App() {
+/**
+ *
+ * NEEDS TO BE SEPARATE FROM APP
+ */
+const SideBoardCard = ({ value, id, total, setTotal, show }) => {
+  // Allows sideboard cards to affect total
+  // REMOVE SIDE CARD FROM BOARD ONCE USED
+  function useSideCard() {
+    if (total + value < 0) {
+      console.log("Cannot bring total below zero!");
+    } else {
+      setTotal(total + value);
+      show;
+    }
+  }
+  return (
+    <button id={id} onClick={() => useSideCard()} className="sideCard">
+      {value}
+    </button>
+  );
+};
+
+/**
+ *
+ * NEEDS TO BE SEPARATE FROM APP
+ */
+const SideBoard = ({ sideBoard, setTotal, total, show }) => {
```

Comment:
Ideally you would want to put new components in their own files to make it a bit easier to organize what is being used by what.